### ISSUE TITLE
Shall we have an organisation of all other crates related to this one?

### ISSUE THREAD

**Issue by NobodyXu** [jonhoo/openssh-rs#40]
Currently `openssh` depends on a few crates maintained by me, including `openssh-mux-client` and `ssh_format`, however these crates only have one single maintainer thus the bus factor is high.

![[xkcd dependency img](https://m.xkcd.com/2347/)](https://imgs.xkcd.com/comics/dependency.png)

Also, since I was the only maintainer, these crates don't get enough people reviewing the code.

**Comment by jonhoo** [comment]
Good point! `openssh-rs` itself has the same problem. I've created an org here: https://github.com/openssh-rust and invited you :)

**Comment by NobodyXu** [comment]
@jonhoo The invitation is cancelled, can you send the link again?

**Comment by jonhoo** [comment]
@NobodyXu done!

**Comment by NobodyXu** [comment]
Thanks!

I finally joined in the organisation!

**Comment by NobodyXu** [comment]
I am gonna move my openssh related crates into the org

**Comment by NobodyXu** [comment]
I've transferred `ssh_format` (core dependency for both `openssh-mux-client` and yet to merge sftp support) into the org.

**Comment by NobodyXu** [comment]
I've finished the transferring.

Now, openssh-mux-clilent, openssh-sftp-protocol and openssh-sftp-client is in the org.

**Comment by NobodyXu** [comment]
Cheers!

Thank you for giving me maintainer privileges for this crate!

**Comment by jonhoo** [comment]
Absolutely well-deserved! Now keep in mind that with great power comes great responsibility — I'll expect you to apply as much rigor to reviews as I've done to yours :)

**Comment by NobodyXu** [comment]
Sure, I will keep that in mind!

**Comment by NobodyXu** [comment]
I figured out that we also need to manually update `Cargo.toml` in all the crates that are moved, including this one.

**Comment by NobodyXu** [comment]
Aha, and also README.md

**Comment by NobodyXu** [comment]
Done.
